azure-pipelines-core.yml

```
@@ -0,0 +1,42 @@
+name: XRTK-$(System.PullRequest.PullRequestNumber)$(Rev:.r)
```

I'd like to be able to fallback to a date if no Pull Request number is available. Is that possible?

---

absolutely
```suggestion
- ${{ if eq(variables[build.reason], 'IndividualCI') }}:
  name: XRTK-$(Date:yyyyMMdd)$(Rev:.r)
- ${{ if nq(variables[build.reason], 'IndividualCI') }}:
  name: XRTK-$(System.PullRequest.PullRequestNumber)$(Rev:.r)
```

is this format good?

---

yup I think that works for me